ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -66,6 +66,33 @@ def test_tensorflow_Acosh(  # NOQA
     )
 
 
+# ApproximateEqual
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.ApproximateEqual",
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
to generate 2 arrays, add the parameter: `num_arrays=2`

as the types of the arrays for this function have to be the same, please add this parameter too: `shared_dtype=True`